client/stylesheets/_responsive.scss

```diff
@@ -215,6 +261,16 @@
     }
 }
 
+@media (max-width: 776px) {
+    .profileCardLi {
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Selector `profileCardLi` should be written in lowercase with hyphens